src/wrappers/themis/wasm/package-lock.json

```diff
@@ -1,20 +1,20 @@
 {
   "name": "wasm-themis",
-  "version": "0.14.5",
+  "version": "0.14.6",
```

Comment:
do we really need to up wasm-themis' version due to changing dev dependency?

Reply:
i think @ilammy plans to move this hotfix to `release/0.14` branch as well, right?

Reply:
> right?

anakin.jpg

Now that you've mentioned it, this _might_ be something to backport, if more hotfixes are expected. I was aiming mostly to fix the nightly scheduled builds on `master`, and unblock people contributing there. Should this be cherry-picked over to the maintenance branch?

The `package-lock.json` is updated by npm and that is consistent with what `package.json` has. Actually, I want to remove `package-lock.json` from the repo altogether (see #916).

Reply:
Given that `release/0.14` builds aren't really failing, I guess I'll give a pass to picking this there. If Node.js 16.x breaks in the same manner like `master` does:

```
TypeError: Module "file:///home/runner/work/themis/themis/src/wrappers/themis/wasm/tsconfig.json" needs an import assertion of type "json"
    at new NodeError (node:internal/errors:372:5)
    at validateAssertions (node:internal/modules/esm/assert:82:15)
    at defaultLoad (node:internal/modules/esm/load:24:3)
    at ESMLoader.load (node:internal/modules/esm/loader:407:26)
    at ESMLoader.moduleProvider (node:internal/modules/esm/loader:326:22)
    at new ModuleJob (node:internal/modules/esm/module_job:66:26)
    at ESMLoader.#createModuleJob (node:internal/modules/esm/loader:345:17)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:304:34)
    at async Promise.all (index 0)
    at ESMLoader.import (node:internal/modules/esm/loader:385:24)
    at importModuleDynamicallyWrapper (node:internal/vm/module:437:15)
    at formattedImport (/home/runner/work/themis/themis/src/wrappers/themis/wasm/node_modules/mocha/lib/nodejs/esm-utils.js:7:14)
    at Object.exports.requireOrImport (/home/runner/work/themis/themis/src/wrappers/themis/wasm/node_modules/mocha/lib/nodejs/esm-utils.js:48:32)
    at Object.exports.loadFilesAsync (/home/runner/work/themis/themis/src/wrappers/themis/wasm/node_modules/mocha/lib/nodejs/esm-utils.js:103:20)
    at singleRun (/home/runner/work/themis/themis/src/wrappers/themis/wasm/node_modules/mocha/lib/cli/run-helpers.js:125:3)
    at Object.exports.handler (/home/runner/work/themis/themis/src/wrappers/themis/wasm/node_modules/mocha/lib/cli/run.js:374:5)
```

feel free to port this patch there.

Reply:
i thought about cherry picking to `release/0.14` but if you say it's green, then why cherry pick :)